tests/globby.js

```diff
@@ -261,6 +261,23 @@ test('gitignore option and objectMode option', async t => {
 	t.truthy(result[0].path);
 });
 
+test.serial('gitignore option and suppressErrors option', async t => {
+	const fooPath = path.join(temporary, 'foo.tmp');
+	const notignoredPath = path.join(temporary, 'notignored');
+	try {
+		fs.mkdirSync(fooPath);
+		fs.chmodSync(fooPath, 0o000);
+		fs.writeFileSync(notignoredPath, 'notignored', 'utf8');
+		const result = await runGlobby(t, temporary, {gitignore: true, suppressErrors: true});
+		t.is(result.length, 1);
+		t.truthy(result.includes('tmp/notignored'));
```

Comment:
The test will pass even if something throws in this block. Use `t.plan()` to ensure the assertions are executed.

Reply:
I didn't quite get this, as there's no `catch` block, but I removed the cleanup since we're now using a real temporary directory instead of the local `tmp/` subdirectory. Let me know if that works for you.